husky_bringup/launch/flir_ptu/flir_ptu.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+
+  <arg name="port" default="/dev/ttyS1"/>
```

Comment:
It's kind of a sticky situation. flir_ptu.launch is copied originally from the flir_ptu package, put in with our concrete parameters to the install directory. Users can pick what install launch flies to use with env vars.

However, we would need to make PRs to flir_ptu to correctly tell them to set env vars. Or modify the install scripts so that we can set env vars which get pushed down to launch file args.